### PR TITLE
Apigee X Custom Network: Secure Bastion

### PR DESCRIPTION
**Pull request by yuriylesyuk**
What's changed, or what was fixed?

**Fixes:** Bastion host uses separate SA

- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

File: tools/apigee-x-trial-provision/README.md [hunk 1]
```diff
@@ -148,23 +148,64 @@ For extended advise on the topic, including GCP recommended naming convention, s
 
 ### Bastion VM in a default network
 
-As Apigee X instance provisioning is a long-running operation, we would recommend to use your working PC terminal or to provision a bastion VM. Bastion VM is also useful for troubleshooting.
+This is an optional section. You can skip it if you know what you are doing. If it is your first time in GCP, follow these instructions.
 
-We are following Best Security Practices here to use a Bastion host and a Service Account to execute gcloud commands and not to expose operator credentials.
+As Apigee X instance provisioning is a long-running operation, it is recommended to use your working PC terminal or to provision a bastion VM. Bastion VM is also useful for troubleshooting, at it would be able to access private networks IP addresses.
```
[the remark this answers is not in the document]
Comment (danistrebel) [reply tools/apigee-x-trial-provision/README.md:153]: either `a private network's IP addresses` or `private network addresses`

File: tools/apigee-x-trial-provision/README.md [hunk 2]
```diff
@@ -148,23 +148,64 @@ For extended advise on the topic, including GCP recommended naming convention, s
...
+```
+
+1. Create a service account for installation purposes.
+
+Click at the Authorize button when asked.
+
+```sh
+export INSTALLER_SA_ID=installer-sa
+
+gcloud iam service-accounts create $INSTALLER_SA_ID
```
Comment (danistrebel) [inline tools/apigee-x-trial-provision/README.md:179]: either use `--project` or use `gcloud config set project $PROJECT`

File: tools/apigee-x-trial-provision/README.md [hunk 3]
```diff
@@ -148,23 +148,64 @@ For extended advise on the topic, including GCP recommended naming convention, s
...
+    gcloud projects add-iam-policy-binding $PROJECT \
+        --member=serviceAccount:$INSTALLER_SA_ID@$PROJECT.iam.gserviceaccount.com \
+        --role=$r
+done
+```
+
+1. Create a compute instance with installer SA identity that will be used to execute script.
+
+```sh
+gcloud compute instances create bastion \
```
Comment (danistrebel) [inline tools/apigee-x-trial-provision/README.md:197]: ditto

**Review by danistrebel — CHANGES_REQUESTED** [review]

File: tools/apigee-x-trial-provision/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (yuriylesyuk) [reply tools/apigee-x-trial-provision/README.md:153]: addressed.

File: tools/apigee-x-trial-provision/README.md (hunk 2 above)
Comment (yuriylesyuk) [reply tools/apigee-x-trial-provision/README.md:179]: fixed: standardised on set project

File: tools/apigee-x-trial-provision/README.md (hunk 3 above)
Comment (yuriylesyuk) [reply tools/apigee-x-trial-provision/README.md:197]: ditto

**Review by danistrebel — APPROVED** [review]
LGTM

**Pull request merged**